VHAR-6588 Tilaajan tarkastusajot erikseen näkyviin

maaritlaine opened the pull request:
VHAR-6588, VHAR-6563, VHAR-7046, VHAR-6667, VHAR-5372, VHAR-6748

maaritlaine committed:
Muuta merkintä merkintäluokaksi.

maaritlaine committed:
Tilaajan tarkastukset erikseen tilannekuvaan.

maaritlaine committed:
Merge branch 'develop' into VHAR-6588

K1ll3rF0x commented on src/clj/harja/palvelin/palvelut/tilannekuva.clj:
```diff
@@ -369,6 +369,8 @@
           tehtavat (into #{}
                          (map (comp :tehtavat #(konv/array->set % :tehtavat)))
                          rivit)
+          _ (println "*** TEHTÄVÄT " tehtavat)
```
Poista debug printit

maaritlaine committed:
Update tilannekuva.clj

Poista kommentti

maaritlaine committed:
Rename V1_950__.sql to V1_961__.sql

maaritlaine committed:
Merge branch 'develop' into VHAR-6588

maaritlaine committed:
Rename V1_961__.sql to V1_979__.sql

maaritlaine committed:
Merge branch 'develop' into VHAR-6588

maaritlaine committed:
Merge branch 'develop' into VHAR-6588

maaritlaine committed:
Merge branch 'develop' into VHAR-6588

K1ll3rF0x committed:
Nimeä migraatio uudelleen

K1ll3rF0x committed:
Korjaa viimeisin_migraatio scriptin shebang
* Scriptissä käytetään bashin komentoja, vaikka shebangissa oli
sh-viittaus

K1ll3rF0x committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-6588

K1ll3rF0x committed:
Hae tarkastajan organisaation nimi tarkastusreittiä klikatessa

K1ll3rF0x committed:
Korjaa bugi tarkastusajojen kartalle haun filtteröinnissä
* Kartan click-handlerin puolelta annetaan 'tekija' parametriksi, mutta
varsinaisessa tarkastusreittien haun SQL:ssä ei filtteröity hakua
tekijän perusteella lainkaan

K1ll3rF0x committed:
Tarkastustyyppi 'tilaajan laadunvalvonta' -> 'tilaajan-laadunvalvonta'

K1ll3rF0x committed:
Käytä 'tilaajan-laadunvalvonta' tyyppiä myös laadunvalvonta/tarkastukset
karttahauissa
* Erotellaan kartan infopaneeliin selkeämmin tilaajan tekemät
tarkastukset vs urakakoitsijan tekemät tiestötarkastukset

K1ll3rF0x committed:
Lisää Tilannekuvan suodattiimiin erikoistapaus 'tilaajan laadunvalvonta'
valinnalle
* Suodatetaan tarkastukset lisäksi organisaation tyypin perusteella
riippuen löytyykö 'tilaajan-laadunvalvonta' suodattimien valinnoista
vai ei.

K1ll3rF0x committed:
Muotoile kyselyn rivit paremmin

K1ll3rF0x committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-6588

K1ll3rF0x committed:
Lisää tiestötarkastusraporttiin sarakkeet tekijälle ja organisaatiolle

K1ll3rF0x committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-6588

K1ll3rF0x committed:
Korjaa tiestötarkastusraportin testit

K1ll3rF0x committed:
Muuta tarkastustyyppi 'tilaajan-laadunvalvonta' takaisin 'tilaajan
laadunvalvonta' nimiseksi
* Tilannekuvan domain koodissa tilaajan-laadunvalvonta symbolilla on
viitattu sekä tarkastustyyppi enumiin (joka tässä oli
'tilaajan-laadunvalvonta') ja suoritettavatehtava enumiin, joka tulee
urakoitsijoiden järjestelmästä arvolla 'tilaajan laadunvalvonta'.
Ristiriitojen takia oli pakko muuttaa tarkastustyyppi takaisin
vastaamaan suoritettavan tehtävän nimeä.
* Tarkastustyyppi muutetaan valitettavasti aina backendissä keywordiksi,
joten keyword viittaukset joutuu toistaiseksi tekemään tyylillä (keyword
"tilaajan laaduntarkastus").

K1ll3rF0x committed:
Korjaa tarkastuksien ja tilannekuvan testit

K1ll3rF0x committed:
Lisää organisaatio ja 'tilaajan laadunvalvonta'-tyyppi tienäkymän
kyselyyn

jpluomajoki-solita reviewed (approved):
Koodi näyttää hyvältä.

K1ll3rF0x committed:
Merge branch 'develop' into VHAR-6588

K1ll3rF0x committed:
Merge branch 'develop' into VHAR-6588
